tasks/bundle.js

```diff
@@ -64,12 +64,13 @@ function collectResources(filepath, html, templateName, done) {
     imgs.push.apply(imgs, srcsetimgs);
     imgs.forEach(function(imgpath) {
       if (imgpath) {
-        let imgFile = imgpath.split('img/')[1];
+        let imgFile = imgpath.split('img' + path.sep)[1];
         const dest = `.archive/${templateName}/img/${imgFile}`;
         fs.copySync(imgpath, dest);
       }
     });
-    let pathToTmpl = filepath.replace(/.*templates\/(.*)/, '\$1');
+    const pathToTmplRegex = new RegExp('.*templates\\' + path.sep + '(.*)');
```

Comment:
I don't believe this is correct.

In the previous implementation, with regex literal, the `\/` is escaping the `/` so it isn't seen as the end of the regex.

The intent isn't to have a literal `\/` or `\\` in the string, which this change will do cause.

Reply:
It's quite confusing, thanks for allowing me the opportunity to explain. In this situation, `\\` is actually `\` escaping `\` so that it doesn't cause the `'` to be escaped. This `\` is then used to escape the path segment separator that `path.sep` returns as both (`\` and `/`) are characters that need to be escaped in JS strings. It took me a while to remember why I did this. 😉